.gitignore

```diff
@@ -13,4 +13,6 @@ android/
 demo/
 
 pubspec.lock
-doc/
\ No newline at end of file
+doc/
+
+.fvm
```

Comment:
Please don't add your custom .gitignore entries in this pull request, either remove them or propose such changes in a different pull request.